fuseki/3.6.0/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM tomcat:8.5.29-jre8-alpine
+
+ENV FUSEKI_VERSION=3.6.0 \
+FUSEKI_BASE=/shared/fuseki-data \
+DEBUG_PORT=5009 \
+SLF4J_VERSION=1.7.25 \
+PASS_PUBLIC_PROTO=https \
+PASS_PUBLIC_HOST=pass
+
+EXPOSE 8080
+EXPOSE ${DEBUG_PORT}
+
+COPY bin/* /bin/
+
```

Comment:
RUN chmod 700 /bin/wait_for_tomcat.sh

Reply:
I still can't explain how this is able to work for docker-for-windows, but a fix is pushed